spyder/utils/help/tutorial.rst

```diff
@@ -423,23 +397,30 @@ We can then use the variables ``x``, ``y``, for example like this:
 Shortcuts for useful functions
 ##############################
 
-- ``F5`` executes the current file
+.. note::
+
+   The following are the default shortcuts; however, those marked with ``*``
+   can be customized through the Keyboard shortcuts tab in the Preferences.
+   Also, macOS users should substitute ``Command`` for ``Ctrl``,
+   and ``Option`` for ``Alt``.
+
+- ``F5``\* executes the current file.
 
-- ``F9`` executes the currently highlighted chunk of code: this is very useful
+- ``F9``\* executes the currently highlighted chunk of code; this is very useful
   to update definitions of functions (say) in the console session without
-  having to run the whole file again. If nothing is selected ``F9`` executes
+  having to run the whole file again. If nothing is selected, ``F9``\* executes
   the current line.
 
-- ``Tab`` auto-completes commands, function names, variable
-  names, methods in the Console (both Python and IPython) and in the
-  Editor. This feature is very useful, and should be used
-  routinely. Do try it now if auto-completion is new to you. 
+- ``Tab``\* auto-completes commands, function names, variable
+  names, and methods in the console and the Editor.
```

Comment:
Editor should be lower case.

Reply:
Looking at other examples below, "editor" there is also spelled "Editor", i.e. upper case. From a purely language (english) point of view, there is no reason for spelling editor with an uppercase E. One could take the view that the "Editor" is an important part of Spyder, and for that reason it should be written with capital "E". In that case, should we do it consistently, and also capitalise "Console"?

Again, nothing major, but I wanted to point out the slight inconsistency.

Reply:
> In that case, should we do it consistently, and also capitalise "Console"?

This is a very good point, and exactly what I've done in the followup #7659 as well as the rest of our docs. As I've specified in our docs style guide (to be formally published in full shortly), the Editor, the Console, the Variable Explorer and the Help pane are proper nouns, i.e. the proper names of core modules of Spyder, and thus they should be capitalized and are also in a different font/style (``:guilabel:`Editor` ``) to indicate they are the names of key GUI elements.

Reply:
Okay, thanks.